### ISSUE TITLE
Behaviour of `read-example` not consistent with doc string

### ISSUE THREAD

**Issue by rmoehn** [clojure-grimoire/lib-grimoire#22]
The [doc string](https://github.com/clojure-grimoire/lib-grimoire/blob/develop/src/grimoire/api.clj#L252-L258) of `grimore.api/read-example` says that it will return a sequence of `(version, example)` tuples. `grimoire.api/read-example`, however, delegates to the multifn `grimoire.api/-read-example`, which [doesn't say](https://github.com/clojure-grimoire/lib-grimoire/blob/develop/src/grimoire/api.clj#L114-L117) anything about a sequence of tuples, but says it will return a single string (wrapped in an `Either`).

The [implementation](https://github.com/clojure-grimoire/lib-grimoire/blob/master/src/grimoire/api/fs/read.clj#L165-L171) does exactly that. I wonder where this sequence-of-tuples thing comes from. Is it the actually desired behaviour or a plan that was abandoned?

**Comment by arrdem** [comment]
So this is all a product of design failures in being the single user and developer for the longest time. Originally, there was no [`list-example`](https://github.com/clojure-grimoire/lib-grimoire/blob/master/src/grimoire/api.clj#L237) implementation, There was only [`read-example`](https://github.com/clojure-grimoire/lib-grimoire/blob/master/src/grimoire/api.clj#L251) and the implementations thereof implicitly used the version enumeration features to find examples on earlier versions. In order to implement the new search API such as it is, I had to add `read-example`.

So basically this was once behavior which I documented and soforth, then decided was silly and backed off of in favor of the current behavior and forgot to redocument.

**Comment by rmoehn** [comment]
I see. Sent a PR.

By the way, wouldn't it make sense to set the default branch to `develop`?

**Comment by arrdem** [comment]
Yes it would. I didn't know you could do that :stuck_out_tongue:

**Comment by arrdem** [comment]
Donezo.
